record/record.go

```diff
@@ -391,6 +460,17 @@ func (w *Writer) Next() (io.Writer, error) {
 	return singleWriter{w, w.seq}, nil
 }
 
+// LastRecordOffset returns the offset in the underlying io.Writer that the last record was written
```

Comment:
I spent a good hour trying to figure out how to word this in a way that was coherent. This is much better.